src/lib/dialog/dialog-config.ts

```diff
@@ -27,6 +27,36 @@ export interface DialogPosition {
   right?: string;
 }
 
+/** Configurable options for `mat-dialog-container`. */
+export interface DialogGlobalOptions {
```

Comment:
You don't have to duplicate all of this code. The config is a `MatDialogConfig`.